rwgame/debugstate.cpp

```diff
@@ -261,12 +261,9 @@ Menu* DebugState::createAIMenu()
 
 #define SPAWN_FOLLOWER(name, id) \
 	m->addEntry(Menu::lambda("Add " name " Follower", [=] { \
-		auto spawnPos = player->getPosition() + player->getRotation() * glm::vec3(0.f, 1.f, 0.f); \
-		auto follower = game->getWorld()->createPedestrian(id, spawnPos); \
-		jumpCharacter(game, follower, spawnPos); \
-		follower->controller->setGoal(CharacterController::FollowLeader); \
-		follower->controller->setTargetCharacter(player); \
+		spawnFollower(id); \
```

Comment:
With this only being a single line you should call this directly. Alternatively I think you should consider using a lambda function.
C++ and macros are meh..